pkg/pillar/types/global.go

```diff
@@ -223,6 +228,7 @@ var GlobalConfigMinimums = GlobalConfig{
 	VdiskGCTime:         60,
 	DownloadRetryTime:   60,
 	DomainBootRetryTime: 10,
+	RktGCGracePeriod:    30,
```

Comment:
You need to add code to EnforceGlobalConfigMinimums for the minimum.

Reply:
Ah yes.. missed that.. Done.. Added.